R/utils.R

```diff
@@ -202,48 +202,48 @@ parse_journal = function(xml_source, outcome, hits, multiple = FALSE,
 #' @param api_answer xml API answer
 #'
 #' @return Returns a data frame with the following columns:
-#' * `romeoid`     {`integer(1)`}\cr{}
+#' * `romeoid`     \[`integer(1)`\]\cr{}
 #'                 the internal index of the publisher in the SHERPA/RoMEO
 #'                 database
-#' * `publisher`   {`character(1)`}\cr{}
+#' * `publisher`   \[`character(1)`\]\cr{}
 #'                 the name of the publisher
-#' * `alias`       {`character(1)`}\cr{}
+#' * `alias`       \[`character(1)`\]\cr{}
 #'                 if applicable an alternative name of the publisher or the
 #'                 name of the specific publishing branch
-#' * `romeocolour` {`character(1)`}\cr{}
+#' * `romeocolour` \[`character(1)`\]\cr{}
 #'                 a colour assigned by the database that reflects the default
 #'                 policies of the publisher
-#' * `preprint`    {`character(1)`}\cr{}
+#' * `preprint`    \[`character(1)`\]\cr{}
 #'                 is the preprint (not reviewed) archivable?
-#' * `postprint`   {`character(1)`}\cr{}
+#' * `postprint`   \[`character(1)`\]\cr{}
 #'                 is the postprint (reviewed but not formatted) archivable?
-#' * `pdf`         {`character(1)`}\cr{}
+#' * `pdf`         \[`character(1)`\]\cr{}
 #'                 is the publisher's version (reviewed and formatted)
 #'                 archivable?
 #'
 #' @keywords internal
 #'
 #' @importFrom httr content http_error
 #' @importFrom xml2 xml_text xml_find_all xml_find_first xml_attr
-parse_publisher = function(xml_source, outcome, hits) {
+parse_publisher <- function(xml_source, outcome, hits) {
 
-  romeoid     = xml_attr(xml_find_all(xml_source, "//publisher"), "id")
+  romeoid     <- xml_attr(xml_find_all(xml_source, "//publisher"), "id")
 
-  publisher   = xml_text(xml_find_all(xml_source, "//name"))
+  publisher   <- xml_text(xml_find_all(xml_source, "//name"))
 
-  alias       = xml_text(xml_find_all(xml_source, "//alias"))
-  alias       = ifelse(alias == "", NA_character_, alias)
+  alias       <- xml_text(xml_find_all(xml_source, "//alias"))
+  alias       <- ifelse(alias == "", NA_character_, alias)
```

Comment:
I can't test it right now but couldn't this be simplified to `alias[alias==""] <- NA`?